src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -247,8 +232,28 @@ def _service_rm(self, cmd):
         svc_type = cmd['svc_type']
         svc_id = cmd['svc_id']
 
-        completion = self._oremote("remove_stateless_service", svc_type, svc_id)
-        self._wait([completion])
+        completion = self.remove_stateless_service(svc_type, svc_id)
+        self._orchestrator_wait([completion])
+        return HandleCommandResult(rs="Success.")
+
+    def _service_action(self, cmd):
```

Comment:
Could you add a simple test to `qa/tasks/mg/test_orchestrator_cli` that executes this code? Doesn't need to perform any action.